public/app/plugins/datasource/stackdriver/partials/query.aggregation.html

```diff
@@ -1,46 +1,56 @@
 <div class="gf-form-inline">
   <div class="gf-form">
     <label class="gf-form-label query-keyword width-9">Aggregation</label>
-    <div class="gf-form-select-wrapper gf-form-select-wrapper--caret-indent">
-      <gf-form-dropdown model="ctrl.target.aggregation.crossSeriesReducer" get-options="ctrl.aggOptions" class="gf-form width-12"
-        disabled type="text" allow-custom="true" lookup-text="true" css-class="min-width-12" on-change="refresh()"></gf-form-dropdown>
-    </div>
+    <metric-select
+      onChange="ctrl.handleAggregationChange"
+      value="ctrl.target.aggregation.crossSeriesReducer"
+      options="ctrl.aggOptions"
+      placeholder="'Select Aggregation'"
+      className="'width-15'"
+      variables="ctrl.templateSrv.variables"
+      group-name="'Aggregations'"
```

Comment:
this property does not exist any more right?

Reply:
Correct! fixed now